airconsole-1.9.0-beta.js

```diff
@@ -1566,3 +1567,25 @@ window.addEventListener('unhandledrejection', function(e) {
     }
   });
 });
+
+
+/**
+ * The devices environment. Only available on the screen device.
+ * Please visit the {@link https://developers.airconsole.com/#!/guides/multiplayer Multiplayer} guide to see how to use this from onDeviceStateChange or through airconsole.devices[AirConsole.SCREEN].environment.id
+ * @typedef {object} AirConsoleScreenEnvironment
+ * @property {string} id - Identifier of the environment this screen is in. Where possible this is a specific physical
```

Comment:
Isn't that what we call Client id (App + Version part)? Could be useful to show an example value.

Reply:
No, this is unrelated to the client. It is the physical environment.
For cars for example this is an identifier that is specific to this exact car and it is meant to be the same for all screens in that car.

For anyone but car it for the moment is always `airconsole`.